src/main/scala/scorex/core/api/http/PeersApiRoute.scala

```diff
@@ -41,10 +40,13 @@ case class PeersApiRoute(peerManager: ActorRef,
       _.flatMap { con =>
         con.peerInfo.map { peerInfo =>
           PeerInfoResponse(
```

Comment:
There's an ignored test in PeersApiRouteSpec for this endpoint, let's unignore it and test the current implementation
```
  //can't check it cause original node using now() timestamp for last seen field
  ignore should "get connected peers" in {
    Get(prefix + "/connected") ~> routes ~> check {
      status shouldBe StatusCodes.OK
      connectedPeersResp shouldBe responseAs[String]
    }
  }
```
other than that - LGTM